tsconfig.docs.json

```diff
@@ -0,0 +1,24 @@
+{
+  "extends": "./tsconfig.json",
+  "include": ["./packages/cli/src"],
```

Comment:
This should include all packages. Is that out of scope for this PR?

Reply:
The goal of this PR was to give the first reasonable proposal for these docs. I wasn't sure, if we should have all packages here, I'll add them :).

Reply:
I think we want to include at least the autogenerated APIs of all packages! Doesn't need to happen in this PR though.